src/pages/home/index.tsx

```diff
@@ -15,12 +16,14 @@ export default class Home extends Component {
                     <span className={styles.light}>Support local charities in Guelph</span> <br />
                     without opening your wallet
                 </h2>
-                <div className={styles.buttons}>
-                    <Button>Currently Donating: <b>14</b></Button>
-                    <Button>Donations to Date <b>345</b></Button>
-                    <Button>Total Hashes <b>{(13256475).toLocaleString()}</b></Button>
-                    <Button color='white'>Login</Button>
-                    <Button color='dark'>Sign Up</Button>
+                <div className={styles.buttongroup}>
+                    <TextBox className={styles.groupelement}>Currently Donating &nbsp; <b>14</b></TextBox>
```

Comment:
Instead of adding non breaking spaces add a class to the b tag with some padding or something

It could be something like this

```tsx
<TextBox className={styles.groupelement}>
    Currently Donating <b className={styles.value}>14</b>
</TextBox>
```
```scss
.value {
    padding-left: 2ch;
}
```